src/pages/rest/b2b/roles.md

```diff
@@ -0,0 +1,1214 @@
+---
+title: Manage company roles
+description: Create and assign user roles for B2B users
+ee_only: True
+---
+
+import * as Vars from '../../../data/vars.js';
+
+# Manage company roles
+
+Within a company, customers may have different job roles, levels of responsibility, and access to information about their company. <Vars.sitedatavarb2b/> defines several types of system resources, and the Company Admin (or an integration that operates on the behalf of the Company Admin) grants or denies access to these resources using company roles. The Company Admin has access to all resources.
+
+<Vars.sitedatavarb2b/> defines the following types of resources:
```

Comment:
The var seems to be affecting the font size of the paragraph. I noticed it in [`company-structures.md`](https://github.com/AdobeDocs/commerce-webapi/pull/19/files#diff-2ac142ca973bf40f8745a5debccc8de4014862b9ac160d15aa8ad9e8d923f681R11) as well. It's not a show-stopper, but it's odd.

![Screen Shot 2022-07-14 at 9 35 57 AM](https://user-images.githubusercontent.com/13662379/179008277-f30e95d7-7c1e-4c5d-9312-1e0a37dc0fcc.png)